pupil_src/main.py

```diff
@@ -18,9 +18,23 @@
 
 import launchable_args
 
-default_args = {"app": "capture", "debug": False, "profile": False}
+default_args = {"app": "capture", "debug": False, "profile": False, "version": False}
 parsed_args, unknown_args = launchable_args.parse(running_from_bundle, **default_args)
 
+# app version
+from version_utils import get_version
+
+app_version = get_version()
+if parsed_args.version:
+    running_from = "bundle" if running_from_bundle else "source"
+    version_message = textwrap.dedent(
+        f"""
+        Pupil {parsed_args.app.capitalize()} version {app_version} ({running_from})
+        """
+    ).strip()
```

Comment:
@papr Are you importing textwrap and doing this multiline-string plus `.strip()` only to keep the line-length?
Why not
```python
version_message = (
    f"Pupil {parsed_args.app.capitalize()} version {app_version} ({running_from})"
)
```
Or are there any other things happening that I don't spot currently?
Or if it's still too long:
```python
version_message = (
    f"Pupil {parsed_args.app.capitalize()}"
    f"version {app_version} ({running_from})"
)
```
And now that I think more about it, why not just:
```python
print(
    f"Pupil {parsed_args.app.capitalize()}"
    f"version {app_version} ({running_from})"
)

Reply:
Not sure what I was thinking.